string name
geometry_msgs/Pose pose
float32 fitness_score
